common/build.gradle

```diff
@@ -125,7 +125,7 @@ android {
 
 dependencies {
     //Java 8 - Desugaring
-    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:$rootProject.ext.coreLibraryDesugaringVersion"
+    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:1.0.9"
```

Comment:
Why not use the version in version file?

Reply:
This is a draft because it isn't available yet.